flink-libraries/flink-cep/src/main/java/org/apache/flink/cep/NonDuplicatingTypeSerializer.java

```diff
@@ -190,13 +190,6 @@ public int hashCode() {
 		return typeSerializer.hashCode();
 	}
 
-	private void readObject(ObjectInputStream ois) throws IOException, ClassNotFoundException {
```

Comment:
removing this means that `elementList` and `identityMap` will not be initialised after deserialisation.